sale_timesheet_analytic_propagation/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL-3.0)
+
+{
+    "name": "Sale Timesheet Analytic Propagation",
```

Comment:
> I have doubts about the module name

The same happened to me, @rafaelbn. The confusion comes from these facts:
- On one hand, at OCA the standard is to name the module like a suffix of the Odoo CE module it's extending. In this case, the module is really extending `sale_timesheet`, because [that module is the one that adds SO line and analytic account to tasks](https://github.com/odoo/odoo/blob/d88a11c488225fbbc41c67549a45511fa7e7dddd/addons/sale_timesheet/models/project.py#L471-L472).
- OTOH, the name doesn't make sense at first sight. A clearer one would be `project_task_analytic_propagation`.

My proposal is that we can keep the module name as it is and, instead, change its human-readable name here:

```suggestion
    "name": "Project Task Analytic Propagation",
```

OTOH if you prefer to name it `project_task_analytic_propagation` I'd completely agree too. WDYT @rafaelbn?

Reply:
I agree @yajo @edlopen